spec/factories/exported_plans.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
```

Comment:
This is perhaps for a larger discussion topic, but I've never understood the purpose of this model. It get created when a user exports their plan to PDF,DOCX, etc. but is never used anywhere else. Not sure why we're recording it if we don't even use the info for stats

Reply:
I think @xsrust feels the same way.